js/actions/DeploymentActions.js

```diff
@@ -0,0 +1,59 @@
+var oboe = require("oboe");
```

Comment:
I haven't used `oboe`, but first think I see when looking at their examples:
```
It isn’t what Oboe was built for but it works fine as a simple AJAX library. 
```

Maybe we should pick a library that does AJAX really well, and that it's not an after thought.
I've used superagent in the past, but there's some other ones like [axios](https://github.com/mzabriskie/axios) if we want more of a promise based lib, which I'm all about.